vm/src/obj/objdict.rs

```diff
@@ -309,12 +306,16 @@ fn dict_getitem(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
 }
 
 pub fn create_type(type_type: PyObjectRef, object_type: PyObjectRef, dict_type: PyObjectRef) {
-    (*dict_type.borrow_mut()).payload = PyObjectPayload::Class {
-        name: String::from("dict"),
-        dict: RefCell::new(HashMap::new()),
-        mro: vec![object_type],
-    };
-    (*dict_type.borrow_mut()).typ = Some(type_type.clone());
+    // this is not ideal
```

Comment:
This will eventually be fixed by some form of mappingproxy like object, so that we do not require a python dict for attributes.